src/bento/programData.js

```diff
@@ -0,0 +1,74 @@
+import gql from 'graphql-tag';
+
+const tableTitle = 'Programs';
+
+const icon = {
+  src: 'https://raw.githubusercontent.com/CBIIT/bento-frontend/master/src/assets/program/programIcon.svg',
+  alt: 'Bento program logo',
+};
+
+const externalLinkIcon = {
+  src: 'https://raw.githubusercontent.com/CBIIT/bento-frontend/master/src/assets/program/externalLinkIcon.svg',
+  alt: 'External link icon',
+};
+
+const table = {
+  data: [
+    {
+      field: 'program_acronym',
+      label: 'Program Code',
+      internalLink: '/program/{program_id}',
+    },
+    {
+      field: 'program_id',
+      label: 'Program ID',
+    },
+    {
+      field: 'program_name',
+      label: 'Program Name',
+    },
+    {
+      field: 'start_date',
+      label: 'Start Date',
+    },
+    {
+      field: 'end_date',
+      label: 'End Date',
+    },
+    {
+      field: 'pubmed_id',
+      label: 'PubMed ID',
+      externalLink: 'https://pubmed.ncbi.nlm.nih.gov/{}',
```

Comment:
If you use this check you can keep just link for both internal and external links
function linkIsExternal(linkElement) {
  return (linkElement.substring(0, 1) !== '/');
}